### ISSUE TITLE
Attempting to create an app that already exists

### ISSUE THREAD

**Issue by belosoGrocerKey** [microsoft/fastlane-plugin-appcenter#65]
When trying to upload an app from my CI, I am getting the following error:
The app belongs to an organization that I am a part of.
The app is still a part of HockeyApp.
I see the app `QA-dev` in the list of apps.

Is there something I am missing?

```
Error 403: {"message"=>"Forbidden", "code"=>"Forbidden"}
App with name QA-dev not found, create one?
```

**Comment by evkhramkov** [comment]
@belosoGrocerKey Hello, could you please open your app in App Center and check its name in the url?
![image](https://user-images.githubusercontent.com/27571887/56656206-8c39af00-669d-11e9-852f-b608904bb86a.png)

**Comment by belosoGrocerKey** [comment]
https://appcenter.ms/orgs/<OrgName>/apps/QA-dev

Is the URL (anonimized the Org)

**Comment by belosoGrocerKey** [comment]
Is there a way to get more debug info?
I have my app created from hockey app.
I don't feel I should create a new app.
I have double checked and the app owner name matches the org in the url.
The API token also seems correct. I can't see what I am missing.

**Comment by evkhramkov** [comment]
That doesn't look like a plugin issue then. Could you please reach out to intercom support in App Center (blue button in the right bottom corner)? We'll need to check your logs to understand why this app is not available for you from plugin

**Comment by belosoGrocerKey** [comment]
Contacting them. Thank you for your assistance, I'll update here when I get an answer.

**Comment by DennisPan** [comment]
@belosoGrocerKey It's also possible that your token is no longer valid, or lost access to the app. Have you been able to figure out the reason with support?

**Comment by filipkowicz** [comment]
plugin doesnt support orgs - only users - it will create new app within your account - you need to patch it by yourself and/or request new feature here.
Check -> https://github.com/vgrigorjeva/fastlane-plugin-appcenter

**Comment by evkhramkov** [comment]
@filipkowicz thanks for report, #75 will fix that

**Comment by jpeg-ms** [comment]
I believe this is now fixed, since #104 was merged.
